test/integration/utils.ts

```diff
@@ -135,13 +135,7 @@ export function scribble(fileName: string | string[], ...args: string[]): string
     return result.stdout;
 }
 
-const defaultArgsMap = new Map<string, string[]>([
-    [
-        "test/samples/ownable_macro.sol",
-        ["--macro-path", "test/samples/macros/ownable.scribble.yaml"]
-    ],
-    ["test/samples/erc20_macro.sol", ["--macro-path", "test/samples/macros"]]
-]);
+const defaultArgsMap = new Map<string, string[]>([]);
```

Comment:
I do not think there is a need for this variable, as it used for reading only and contains no values.

Reply:
Fixed in 2e5e76b.